### DISCUSSION TITLE
GameCube Adapter not detecting Z input from NSO N64 Controller

### DISCUSSION THREAD

**Discussion by Hyrule-Hero** [darthcloud/BlueRetro#826] [General]
Hello,

I recently purchased the GameCube adapter and I updated to the current 1.9 firmware. While playing Paper Mario on Wii VC, the Z button is not working when it is mapped to Analog L. I also tried to map a button on the dpad to Analog L and it still does not register. Is there anything I can do to make a fix for this?

**Comment by darthcloud** [comment]
Map it to Digital L?
